src/alda/AldaClient.java

```diff
@@ -81,6 +106,11 @@ public static void updateAlda() throws URISyntaxException {
       return;
     }
 
+    // Copy existing alda binary to temp directory
+    if (!Util.moveFile(aldaPath, aldaCopyPath))
+      return;
```

Comment:
Oh -- reading the code more, it looks like `Util.moveFile` will already do that if it fails. 👍